app/views/games/index.html.erb

```diff
@@ -1 +1,21 @@
 <h1>Game Page</h1>
+<div class="container">
+  <div class="panel panel-default">
+    <div class="panel-heading">Available Games</div>
+    <table class="table">
+      <tr>
+        <th>#</th>
+        <th>Player</th>
+        <th>You would be...</th>
+      </tr>
+      <% @available_games.each_with_index do |available_game, i| %>
+        <tr>
+          <td><%= available_game.id %></td>
+          <td><%= (available_game.user_black && available_game.user_black.email) || (available_game.user_white && available_game.user_white.email) %></td>
```

Comment:
Could we move this logic into the model itself under a method like `missing_player_color` or something?  Should help keep the views more streamlined